src/GitHub/Data/Webhooks/Payload.hs

```diff
@@ -73,6 +73,7 @@ instance NFData URL where rnf = genericRnf
 instance FromJSON URL where
   parseJSON = withText "URL" (pure . URL)
 
+-- | Convert Github URL to Text
```

Comment:
You can link types here. When you generate the documentation they will form hyperlinks. For example
```
-- | Converts a 'URL' to 'Text'.
```

Reply:
I agree with you.